frontend/src/features/videos/VideoCard.tsx

```diff
@@ -218,9 +220,18 @@ function VideoCard({
               {video.publication_date}
             </span>
           )}
-          {video.uploader && (
-            <span className={classes.channel}>{video.uploader}</span>
-          )}
+          <Tooltip title="Filter by channel" placement="bottom">
+            <div>
+              {video.uploader && (
+                <Link
+                  to={`/recommendations?uploader=${video.uploader}`}
```

Comment:
The uploader name needs to be URL-encoded here, for example using `encodeURIComponent`, or `URLSearchParams`. Otherwise special characters could lead to invalid query parameters in the URL.

Reply:
However here, the URL is passed directly to the History API, and will update the current URL. An uploader name such `"a&b"` would be injected as `?uploader=a&b` and would be parsed as 2 distincts parameters `uploader` and `b`.  

That's why the parameter needs to be encoded explicitly here ;)

```suggestion
                  to={`/recommendations?uploader=${encodeURIComponent(video.uploader)}`}
```